geopandas/geodataframe.py

```diff
@@ -970,18 +970,59 @@ def iterfeatures(self, na="null", show_bbox=False, drop_id=False):
 
                 yield feature
 
-    def _to_geo(self, **kwargs):
+    def to_geo_dict(self, na="null", show_bbox=False, drop_id=False):
         """
-        Returns a python feature collection (i.e. the geointerface)
-        representation of the GeoDataFrame.
+        Returns a python feature collection representation of the GeoDataFrame
+        as a dictionary with a list of features using the GeoJSON specification.
```

Comment:
```suggestion
        as a dictionary with a list of features based on the ``__geo_interface__``
        GeoJSON-like specification.
```

Strictly speaking, this is not exactly GeoJSON and with the options, it is _like_ a `__geo_interface__` but with the default `show_bbox=False` it does not follow it exactly, so "based on" is better wording imo.